content/00.build/05.start-coding/10.zksync-101/10.hello-zksync.md

```diff
@@ -3,48 +3,196 @@ title: Hello ZKsync!
 description: Learn to deploy smart contracts efficiently in the ZKsync environment.
 ---
 
-Welcome to the ZKsync 101 guide for deploying smart contracts on ZKsync! In this series, we'll walk you through the process
+Welcome to ZKsync 101 for deploying smart contracts on ZKsync! In this series, we'll walk you through the process
 of creating and deploying a simple smart contract that creates a crowdfunding campaign for Zeek. In this section you will learn the following:
```

Comment:
I think `learn the following` doesn't flow well for how the checklist is worded, maybe complete the following or learn how to